backtrace-library/src/main/cpp/backends/backend.cpp

```diff
@@ -82,4 +82,29 @@ void Disable() {
                         "Disable not supported on this backend");
 #endif
 }
+
+bool EnableCrashLoopDetectionBackend() {
```

Comment:
it doesn't make sense to add `Backend` suffix here. It doesn't provide any useful information to the user but adds more confusion. What does the backend mean? I think it would be better if we rename method to `EnableCrashLoopDetection` and we follow the same pattern with the rest of those methods